### PR TITLE
JAPI-35, JAPI-30, Serialization of DelimitedDataOptions and EclRecordInfo

### PR DESCRIPTION
**Pull request by Michael-Gardner**

### CODE REVIEW

File: wsclient/src/main/java/org/hpccsystems/ws/client/utils/DelimitedDataOptions.java [hunk 1]
```diff
@@ -5,7 +5,9 @@
  *
  * Represents variable/delimited field data formats
  */
-public class DelimitedDataOptions
+import java.io.Serializable;
+
+public class DelimitedDataOptions implements java.io.Serializable
 {
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/utils/DelimitedDataOptions.java]: should this class have a generated serialVersionUID?

**Comment by rpastrana** [comment]
Mike, please squash these two commits.

**Comment by Michael-Gardner** [comment]
@rpastrana Squashed and added the unique Id to DelimitedDataOptions

**Comment by rpastrana** [comment]
Looks good.

**Pull request merged**